pkg/autodiscover/autodiscover_operators.go

```diff
@@ -31,16 +31,17 @@ import (
 )
 
 const (
-	istio = "istio-system"
+	istioNamespace = "istio-system"
 )
 
 func findnamespace(oc corev1client.CoreV1Interface) bool {
```

Comment:
I also added a comment about [the name of this function](https://github.com/test-network-function/cnf-certification-test/pull/228#discussion_r898209585). Can you review it, please?

Reply:
sure, tnx for your attention.